MDISforLinux/DRIVERS/BBIS/SMB2BB/DRIVER/NATIVE/bb_smb2.c

```diff
@@ -572,13 +456,11 @@ static int32 SMB2BB_BrdInit( BBIS_HANDLE *h )
     }
     OSS_MemFill(oss, gotsize, (u_int8*)newHdl, 0);
 
-
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,LX_VERSION_I2C_DIFFERS)
     /* init addr_data, we have to probe the whole Bus on a adapter */
     for ( i = 0; i<SMB2_I2C_ADDR_COUNT-1; i++)
-		normal_addr[i] = (SMB2_I2C_START_ADDR+i);
+		  normal_addr[i] = (SMB2_I2C_START_ADDR+i);
```

Comment:
Is this line well indented?

Reply:
No it was not, it was mixing spaces and tabs on the same line. The entire file needs a style refactor to fix indentation. I just force pushed the branch to keep history as initially intended fixing just this line.

Reply:
perfect